lib/single-new-eks-opensource-observability-pattern/graviton-index.ts

```diff
@@ -59,6 +59,12 @@ export default class SingleNewEksGravitonOpenSourceObservabilityPattern {
             "{{ stop enableAdotMetricsCollectionTelemetry }}",
             jsonStringnew.context["adotcollectormetrics.pattern.enabled"]
         );
+        doc = utils.changeTextBetweenTokens(
```

Comment:
Please pull code from main. We had some significant changes.

Reply:
merged with main